src/main/webapp/WEB-INF/tags/student/comments/pagination.tag

```diff
@@ -0,0 +1,13 @@
+<%@ tag description="Student comments course pagination" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<ul class="pagination">
+    <li><a href="${data.previousPageLink}">�</a></li>
+    <c:forEach items="${data.coursePaginationList}" var="courseId">
+    <li
+        class="${courseId == data.courseId ? 'active' : ''}">
+        <a href="${data.studentCommentsLink}&courseid=${courseId}">${courseId}</a>
+    </li>
+    </c:forEach>
```

Comment:
Slight mistake in indentation here.
Also, you can probably combine lines 7 and 8.
If I'm not wrong, this is similar to your previous PR?
They could benefit by using the same class and tag if they're similar enough.